### ISSUE TITLE
[BUG]: DeepFace cannot load images with Unicode (Korean) filenames

### ISSUE THREAD

**Issue by syw1236** [serengil/deepface#1529]
### Before You Report a Bug, Please Confirm You Have Done The Following...

- [x] I have updated to the latest version of the packages.
- [x] I have searched for both [existing issues](https://github.com/serengil/deepface/issues) and [closed issues](https://github.com/serengil/deepface/issues?q=is%3Aissue+is%3Aclosed) and found none that matched my issue.

### DeepFace's version

0.0.96

### Python version

3.11.7

### Operating System

macOS

### Dependencies

absl-py==2.3.1
astunparse==1.6.3
attrs==25.4.0
beautifulsoup4==4.14.2
blinker==1.9.0
certifi==2025.11.12
charset-normalizer==3.4.4
click==8.3.1
deepface==0.0.96
filelock==3.20.0
fire==0.7.1
Flask==3.1.2
flask-cors==6.0.1
flatbuffers==25.9.23
gast==0.6.0
gdown==5.2.0
google-pasta==0.2.0
grpcio==1.76.0
gunicorn==23.0.0
h5py==3.15.1
idna==3.11
iniconfig==2.3.0
itsdangerous==2.2.0
Jinja2==3.1.6
joblib==1.5.2
keras==3.12.0
libclang==18.1.1
lightecc==0.0.3
lightphe==0.0.15
lz4==4.4.5
Markdown==3.10
markdown-it-py==4.0.0
MarkupSafe==3.0.3
mdurl==0.1.2
ml_dtypes==0.5.4
mpmath==1.3.0
mtcnn==1.0.0
namex==0.1.0
numpy==2.2.6
opencv-python==4.12.0.88
opt_einsum==3.4.0
optree==0.18.0
packaging==25.0
pandas==2.3.3
pillow==12.0.0
pluggy==1.6.0
protobuf==6.33.1
py==1.11.0
Pygments==2.19.2
PySocks==1.7.1
pytest==7.1.2
python-dateutil==2.9.0.post0
pytz==2025.2
requests==2.32.5
retina-face==0.0.17
rich==14.2.0
six==1.17.0
soupsieve==2.8
sympy==1.12
tensorboard==2.20.0
tensorboard-data-server==0.7.2
tensorflow==2.20.0
termcolor==3.2.0
tf_keras==2.20.1
tomli==2.3.0
tqdm==4.67.1
typing_extensions==4.15.0
tzdata==2025.2
urllib3==2.5.0
Werkzeug==3.1.3
wrapt==2.0.1

### Reproducible example

```Python
from deepface import DeepFace

print("Unicode filename test")

result = DeepFace.verify(
    img1_path="아이유.jpg",
    img2_path="iu2.jpg"
)

print(result)
```

### Relevant Log Output

Exception while processing img1_path

### Expected Result

DeepFace should correctly load and process images even when the filename contains Unicode characters such as Korean.

For example, "아이유.jpg" should be treated the same as "iu.jpg".

### What happened instead?

DeepFace throws an exception when the image filename contains Korean characters.

The same image works perfectly when renamed to an ASCII-only filename such as "iu.jpg".

### Additional Info

I confirmed that the issue comes from the ASCII-only validation in load_image().
I am preparing a Pull Request to remove this restriction and support Unicode filenames safely.
